pkg/hub/api/grpc/storage_backend/client_test.go

```diff
@@ -0,0 +1,189 @@
+package storage_backend_test
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"testing"
+
+	pb "capact.io/capact/pkg/hub/api/grpc/storage_backend"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+)
+
+// #nosec G101
+const secretStorageBackendAddr = "GRPC_SECRET_STORAGE_BACKEND_ADDR"
+
+// This test illustrates how to use gRPC Go client against real gRPC Storage Backend server.
+//
+// NOTE: Before running this test, make sure that the server is running under the `srvAddr` address
+//	and the `provider` is enabled on this server.
+//
+// To run this test, execute:
+// GRPC_SECRET_STORAGE_BACKEND_ADDR=":50051" go test ./pkg/hub/api/grpc/storage_backend -v
```

Comment:
```suggestion
// GRPC_SECRET_STORAGE_BACKEND_ADDR=":50051" go test ./pkg/hub/api/grpc/storage_backend -v -count 1
```

Reason: When I executed this test for the first time with the `dotenv` it passed. Executing it for the second time when only backend was changed to accept AWS only, it also passed and printed _cached_ results, so this will solve this problem.